app/views/admin/accounting/transactions/show.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#transaction_show').html("<%= j render 'show_modal' %>");
```

Comment:
This is not standard. We usually use a Backbone view. We would build upon `transactions_view.coffee` and `transactions_modal_view.coffee`.

Examples of a JavaScript, controller, and template flow (where these work together) are project logs and project groups.